openshiftcontrolplane/v1/types.go

```diff
@@ -204,6 +204,11 @@ type BuildControllerConfig struct {
 	// additionalTrustedCA is a path to a pem bundle file containing additional CAs that
 	// should be trusted for image pushes and pulls during builds.
 	AdditionalTrustedCA string `json:"additionalTrustedCA"`
+
+	// buildCSIVolumes defines whether the instances of DockerBuildStrategy or SourceBuildStrategy
+	// can employ CSI volumes with any BuildVolume they specify.  Currently, this is only
+	// supported on clusters where the TechPreview feature set is enabled.
+	BuildCSIVolumes bool `json:"buildCSIVolumes,omitempty"`
```

Comment:
same alignment issue on this struct.  And same question to @deads2k about whether we'd want to prefix with Enable